modules/local/plot_entity_binding_ratios.nf

```diff
@@ -0,0 +1,43 @@
+process PLOT_ENTITY_BINDING_RATIOS {
+    label 'process_medium_memory'
+
+    conda "conda-forge::r-ggplot2=3.4.2 conda-forge::r-data.table=1.14.8 conda-forge::r-dplyr=1.1.2 conda-forge::r-stringr=1.5.0 conda-forge::r-ggpubr=0.6.0 conda-forge::r-optparse=1.7.3"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-0be74e7b0c2e289bc8098b1491baf4f181012b1c:a1635746bc2c13635cbea8c29bd5a2837bdd7cd5-0' :
+        'biocontainers/mulled-v2-0be74e7b0c2e289bc8098b1491baf4f181012b1c:a1635746bc2c13635cbea8c29bd5a2837bdd7cd5-0' }"
+
+    publishDir "${params.outdir}/figures", mode: params.publish_dir_mode
```

Comment:
Maybe it would be better to have this in `modules.config`, for consistency with other modules

Reply:
Absolutely true, I have no Idea how that ended up in the process 😆